data/companies.json

```diff
@@ -766,8 +766,24 @@
         "instagram": null,
         "linkedin": "https://www.linkedin.com/company/yco-pvt.-ltd./",
         "twitter": null,
-        "youtube": null,
+        "youtube": null
       }
     },   
+    {
+      "name": "BlackBox Technologies Pvt. Ltd.",
+      "establishedAt": 2018,
```

Comment:
The `establishedAt` is a unix timestamp. Please read CONTRIBUTING.md before opening a PR.

Reply:
Oh my bad! I will fix it ASAP. Thanks.

Reply:
@thenishantsapkota I've changed `establishedAt` to unix timestamp.